CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+# Changelog
+Starting in version 4.0.0 we will maintain a changelog
+
+## [4.0.0](https://github.com/actions/stale/compare/v3.0.19...v4.0.0) (2021-06-18)
+
+
+### Features
+
+* **options:** add new options to avoid stale base on comments ([#494](https://github.com/actions/stale/issues/494)) ([1efddcb](https://github.com/actions/stale/commit/1efddcbe9ff0497885b9746a8ea5fa0314286053)), closes [#441](https://github.com/actions/stale/issues/441) [#470](https://github.com/actions/stale/issues/470) [#435](https://github.com/acti
+ons/stale/issues/435) [#390](https://github.com/actions/stale/issues/390)
+* **options:** simplify config by removing skip stale message options ([#457](https://github.com/actions/stale/issues/457)) ([6ec637d](https://github.com/actions/stale/commit/6ec637d238067ab8cc96c9289dcdac280bbd3f4a)), closes [#405](https://github.com/actions/stale/issues/405) [#455](https://github.com/actions/stale/issues/455)
+* **output:** print output parameters ([#458](https://github.com/actions/stale/issues/458)) ([3e6d35b](https://github.com/actions/stale/commit/3e6d35b685f0b2fa1a69be893fa07d3d85e05ee0))
+
+
+### Bug Fixes
+
+* **dry-run:** forbid mutations in dry-run ([#500](https://github.com/actions/stale/issues/500)) ([f1017f3](https://github.com/actions/stale/commit/f1017f33dd159ea51366375120c3e6981d7c3097)), closes [#499](https://github.com/actions/stale/issues/499)
+* **logs:** coloured logs ([#465](https://github.com/actions/stale/issues/465)) ([5fbbfba](https://github.com/actions/stale/commit/5fbbfba142860ea6512549e96e36e3540c314132))
+* **operations:** fail fast the current batch to respect the operations limit ([#474](https://github.com/actions/stale/issues/474)) ([5f6f311](https://github.com/actions/stale/commit/5f6f311ca6aa75babadfc7bac6edf5d85fa3f35d)), closes [#466](https://github.com/actions/stale/issues/466)
+---
+
+### Breaking Changes
+
+??
```

Comment:
```suggestion
* The options `skip-stale-issue-message` and `skip-stale-pr-message` were removed. Instead, setting the options `stale-issue-message` and `stale-pr-message` will be enough to let the stale workflow add a comment. If the options are unset, not comment will be added which was the equivalent of setting `skip-stale-issue-message` at `true`.
* The `operations-per-run` option will be more effective. After migrating, you could face a failed-fast process workflow if you let the default value (30) or set it to a small number. In that case, you will see a warning at the end of the logs (if enabled) indicating that the workflow was stopped sooner to avoid consuming too much API calls. In most cases, you can just increase this limit to make sure to process everything in a single run.
```

Reply:
not->no: "If the options are unset, no comment"